extension/src/Package.ts

```diff
@@ -286,6 +287,20 @@ function uriJoin(directory: vscode.Uri, file: string) {
 
 function findVsixFile(manifest: PackageManifest) {
     if (manifest.files) {
+        if (manifest.osSpecific) {
+            const os = require("os");
```

Comment:
We should probably just `import` os at the top of the file.

Reply:
Done